### ISSUE TITLE
Incorrect Source for EIDs with ASCO abstracts as the source type

### ISSUE THREAD

**Issue by cgrisdale** [genome/civic-client#1325]
Many ASCO-sourced EIDs have irrelevant journal articles listed as Source, or nothing listed.

![Screenshot from 2020-02-06 10-55-14](https://user-images.githubusercontent.com/3093548/73969314-04d9c700-48d0-11ea-9104-43be558358e5.png)

![Screenshot from 2020-02-06 11-06-07](https://user-images.githubusercontent.com/3093548/73969655-b1b44400-48d0-11ea-899a-5c398a15193d.png)

**Comment by susannasiebert** [comment]
We should check with the submitter to figure out which citation they meant to submit with this.

**Comment by susannasiebert** [comment]
It looks like the title and abstract that got pulled in don't match the asco abstract id. When I execute the same code that scraps this information locally, it fetches the correct info so it's not clear to me what happened here. The only thing I can think of is that ASCO was returning the incorrect information at one point in the past? We should re-fetch all of the information for all of our ASCO sources.

**Comment by susannasiebert** [comment]
We figured out that apparently we were importing data from PubMed for these citation IDs. All of our ASCO sources were imported between August and September of last year. but when I submit source suggestion now for these IDs  on the staging server, it pulls in the correct info. So the underlying bug appears to be fixed but we will need to rerun the scraper to update the data for our ASCO sources.

**Comment by susannasiebert** [comment]
TLDR: https://github.com/griffithlab/civic-server/pull/588 contains a fix for this issue and has been deployed to production. The ASCO sources have been fixed

Long version: There exists a background job called FixPubmedEntriesMissingAuthors that is used to backfill PubMed sources that are missing their author information. The way it determined which PubMed entries to process was a two-step process where we first grabbed all sources that have author information and then substracted that set from all sources. Before adding ASCO sources this would work fine but not that second step would return PubMed sources without authors and ASCO sources. Basically, the initial import would scrape our information just fine but this job was submitting our ASCO sources to be scraped using the PubMed scraper, which is why we were seeing PubMed content for those ASCO sources with matching IDs. The above PR fixes the query. It has been deployed to production so I don't expect this issue to reoccur in the future. Existing ASCO sources have been fixed.
